.node-version

```diff
@@ -0,0 +1 @@
+10.6.0
```

Comment:
Is this something picked up by Travis?

Reply:
I'd prefer to just use .travis.yml, if so.

Reply:
Ah sorry, was my nodenv setting. Wait a minute, I'll remove it.

Reply:
Done.